wiki/Beatmap/Pattern/Mania/Streams/en.md

```diff
@@ -0,0 +1,41 @@
+---
+stub: true
+tags:
+  - streams
+---
+
+# Streams
+
+_For specific stream types unrelated to 4-key, refer to [chordstreams](mania/chordstreams)._
+
+**Streams** are constant notes within the same snap interval that occur for at least one beat, avoiding an occurance of a [minijack](/wiki/Beatmap/Pattern/Mania/Jacks/en.md#Minijack). There are many different types of streams that impose different levels of strain on the player. When stream is used on its own, it's likely referring to a **single note stream**.
```

Comment:
```suggestion
**Streams** are constant notes within the same snap interval that occur for at least one beat, avoiding an occurance of a [minijack](/wiki/Beatmap/Pattern/Mania/Jacks/en.md#Minijack). There are many different types of streams that impose different levels of strain on the player. When a stream is used on its own, it's likely referring to a **single note stream**.
```

Aren't those also called `single streams`?

Reply:
I feel as if it's better to specify the note part since there's terminology which refers to chord sizes, and single streams is not a difficult thing to deduce from single note streams anyways imo.